### ISSUE TITLE
Need to expose a way to access raw JSON for response

### ISSUE THREAD

**Issue by jhump** [fullstorydev/grpcui#2]
There is currently a way to show the raw JSON for the requests that will be sent. Ideally, we could do the same thing for response data (which currently is only rendered in an HTML form).

**Comment by jhump** [comment]
Perhaps even more important is to include not only the field names in the response table but also the field types. This is necessary, for example, to distinguish between regular message types, map types with string keys (particularly when used with keys that are also legal protobuf identifiers), and arbitrary JSON data in the form of `google.protobuf.Value` messages (and related types defined in `google/protobuf/struct.proto`).

**Comment by knipknap** [comment]
That would be such a useful feature to have, especially to create test data. Guess I'll have to hand-craft the expected response from scratch, ouch.

**Comment by nelk** [comment]
Currently when trying to render huge responses, the page freezes up and is unusable, at least in chrome.
If this feature is implemented it should come with the ability to default to a raw json response view to avoid the page trying to render the whole parsed response.

**Comment by jhump** [comment]
So I think the first way to start might be to simply replace the formatted HTML view of the response with a textarea that shows the raw JSON. That will make the UI faster and make the response more useful, since it can be copied.

A future improvement might be an option here, to toggle from JSON to a structured HTML view, but that won't likely be useful unless it could show more than just the JSON structure -- like if it included information about field types and maybe even allowed accessing things like field definitions (options, comments, etc).

**Comment by theMPatel** [comment]
so kinda like:
![Screen Shot 2020-11-11 at 11 36 53 AM](https://user-images.githubusercontent.com/32933108/98838416-3d37a400-2412-11eb-91d9-6aeef5646671.png)

**Comment by jhump** [comment]
@theMPatel, hawt! Got a PR for that?

That seems functionally better than the "pretty" HTML form that's there now. Sadly, it is often just a hindrance.

**Comment by theMPatel** [comment]
I have a branch but I can't seem to push it to remote :(

```sh
>>> gpsup

ERROR: Permission to fullstorydev/grpcui.git denied to theMPatel.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by theMPatel** [comment]
ok still learning the whole fork/commit model:  #100

**Comment by jhump** [comment]
Resolved in #100

**Comment by mrinal-sharma** [comment]
@jhump @thempatel I want to use this feature, how can I use this?
